README.adoc

```diff
@@ -88,13 +85,10 @@ Consider an example that you want to add `--max-scale=5` and `--min-scale=1`, th
   uses: redhat-actions/kn-service-deploy@v1
   with: 
     service_name: fruits-app
-    service_namespace: ${{ env.NAMESPACE }}
     container_image: "${{ steps.push-tag-to-quay.outputs.registry-path }}"
     service_params: >
       --max-scale=5
       --min-scale=1
 ----
 
-The above action configuration will be interpreted in to `kn` command like:
-
-For a complete action example check https://github.com/redhat-actions/kn-service-deploy/blob/main/.github/workflows/example.yml[kn-service-action-example].
+For a complete action example check https://github.com/redhat-actions/kn-service-deploy/blob/main/.github/workflows/example.yml[here].
```

Comment:
You can use relative links in github repositories to simplify the link and prevent breaking if the repo or branch changes:

`https://github.com/redhat-actions/kn-service-deploy/blob/main/.github/workflows/example.yml`
->
`./.github/workflows/example.yml`

Reply:
That makes sense, will fix.